lms/services/blackboard_api.py

```diff
@@ -1,18 +1,5 @@
-import re
```

Comment:
The `BlackboardAPIClient` now fits in a single `services/blackboard_api.py` files. It's no longer necessary to have a `services/blackboard_api/` subdir with `service.py` and `_schemas.py` files because the schemas are no longer part of the service: they've moved into the views.